recipes/piawka/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "piawka" %}
+{% set version = "0.7.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../
+  url: https://github.com/novikovalab/piawka/archive/refs/tags/0.7.10.tar.gz
+  sha256: 511859b69525db7dc988ae55556e6fd31e2709f0e3a4de404f87abfadb97a273
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('piawka', max_pin="x") }}
+  script: #replace shebangs
+    - set -x
+    - pwd
+    - mkdir -p $PREFIX/bin
+    - polyglot1='#!/bin/sh'; polyglot2='"exec" "mawk" "-f" "$0" "$@" && 0 {}'
+    - for f in $SRC_DIR/scripts/*; do t="$PREFIX/bin/$(basename $f)"; head -n 1 $f | grep awk && ( echo "$polyglot1" > $t && echo "$polyglot2" >> $t ); cat $f >> $t; done
+    - chmod +x $PREFIX/bin/*
+    - mkdir -p $PREFIX/test
+    - cp $SRC_DIR/examples/{minitest.vcf.gz,groups.tsv} $PREFIX/test
+
+requirements:
+  run:
+    - mawk =1.3.4
+    - parallel
+    - bash
+    - tabix
+    
+test:
+  commands:
+    - "piawka $PREFIX/test/groups.tsv <(zcat $PREFIX/test/minitest.vcf.gz) > $PREFIX/test/test.out" 

```

Comment:
This test might be overkill. Usually we use something like checking for the version or `--help` or something like that.
See https://bioconda.github.io/contributor/guidelines.html#tests

Reply:
revised